cmd/tikv-ctl/src/main.rs

```diff
@@ -519,7 +519,15 @@ fn main() {
                 Cmd::Cluster {} => {
                     debug_executor.dump_cluster_info();
                 }
-                Cmd::ResetToVersion { version } => debug_executor.reset_to_version(version),
+                Cmd::ResetToVersion { version, force } => {
+                    // This command is deprecated which cannot make sure transaction correctness.
```

Comment:
or can we just remove it

Reply:
Do we also need to remove server-related code?

Reply:
If no others use, yes